hack/build_cli.sh

```
@@ -16,7 +16,7 @@ do
     fi
     GOOS=${os} GOARCH=${arch} go build -o /tmp/rosa_${os}_${arch} ./cmd/rosa
     mv /tmp/rosa_${os}_${arch} rosa${extension}
-    zip releases/rosa_${os}_${arch}.zip rosa${extension}
+    tar -czf releases/rosa_${os}_${arch}.tar.gz rosa${extension}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)